apps/registration/api.py

```diff
@@ -44,46 +40,34 @@ def create(self, request):
                 status=status.HTTP_400_BAD_REQUEST,
             )
 
-        # send activation email to user
         current_site = get_current_site(request)
-        message = render_to_string('acc_active_email.html', {
-            'user': user,
-            'domain': current_site.domain,
-            'uid': urlsafe_base64_encode(force_bytes(user.pk)),
-            'token': account_activation_token.make_token(user),
-        })
-        mail_subject = 'Activate your account!'
-        to_email = user.email
-        email = EmailMultiAlternatives(
-            mail_subject,
-            message,
-            to=[to_email]
+        send_template_email(
+            [user.email],
+            'Activate your account!',
+            'acc_active_email.html',
+            {
+                'user': user,
+                'domain': current_site.domain,
+                'uid': urlsafe_base64_encode(force_bytes(user.pk)),
+                'token': account_activation_token.make_token(user),
+            })
+
+        # send intro email
+        send_template_email(
+                [user.email],
+                'Thanks for applying!',
+                'intro_email.html',
+                {},
+                settings.INTRO_EMAIL_FROM,
+                settings.INTRO_EMAIL_DELAY
```

Comment:
nit: put a comma at the end of the last parameter even though you don't need to - this makes it easier to add / move around parameters